### ISSUE TITLE
Chromatic 5.6.2 Breaks in CircleCI 

### ISSUE THREAD

**Issue by hamatoyogi** [chromaui/chromatic-cli#281]
Running V5.6.2 breaks on Circle CI with this script:
```
#!/bin/bash -eo pipefail
if [ "${CIRCLE_BRANCH}" != "main" ];
  then
    yarn chromatic --project-token=${CHROMATIC_PROJECT_TOKEN} --exit-zero-on-changes
else
  yarn chromatic --project-token=${CHROMATIC_PROJECT_TOKEN} --auto-accept-changes
fi

yarn run v1.22.4
$ npx chromatic --project-token ********** --project-token=********** --exit-zero-on-changes
Unhandled promise rejection: The flag --projectToken can only be set once.
Error: The flag --projectToken can only be set once.
    at validateFlags (/home/circleci/project/node_modules/chromatic/node_modules/meow/index.js:101:10)
    at meow (/home/circleci/project/node_modules/chromatic/node_modules/meow/index.js:213:2)
    at parseArgs (/home/circleci/project/node_modules/chromatic/bin/lib/parseArgs.js:4:39)
    at Proxy.main (/home/circleci/project/node_modules/chromatic/bin/main.js:32:67)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Exited with code exit status 1
CircleCI received exit code 1
```

Works fine on v5.5.0.

**Comment by bodograumann** [comment]
This is the same issue as #235. To recap:
- meow 8.1.1 introduced a breaking change which caused this issue
- meow 8.1.2 roled back the change and post-poned it to 9.0.0
- now chromatic 5.6.2 has updated to meow 9.0.0 and thus also introduced the problem again

**Comment by tmeasday** [comment]
@bodograumann this seems like a different issue to #235? 

@hamatoyogi it looks like your `package.json` already sets the `--project-token` flag? So you could simplify your script to:

```
if [ "${CIRCLE_BRANCH}" != "main" ];
  then
    yarn chromatic --exit-zero-on-changes
else
  yarn chromatic --auto-accept-changes
fi
```

**Comment by tmeasday** [comment]
I suspect we should revert the upgrade until the next major version of `chromatic` though.

**Comment by hamatoyogi** [comment]
> @bodograumann this seems like a different issue to #235?
> 
> @hamatoyogi it looks like your `package.json` already sets the `--project-token` flag? So you could simplify your script to:
> 
> ```
> if [ "${CIRCLE_BRANCH}" != "main" ];
>   then
>     yarn chromatic --exit-zero-on-changes
> else
>   yarn chromatic --auto-accept-changes
> fi
> ```

You might be right @tmeasday. I'll try it out like that.
Then again, why would it work like that in 5.5.0 and no in 5.6.2? 
In any way, it seems like i would wait with the upgrade nonetheless. 

Thanks for the replies!

**Comment by ndelangen** [comment]
I'll be releasing a patch version today with a meow downgrade

**Comment by bodograumann** [comment]
@tmeasday Oh damn, you're right. I did not read this issue thoroughly enough when following the suggestion in the #235.
might be that both symptoms are caused by the meow upgrade, but I can’t be sure about that.

**Comment by ndelangen** [comment]
I released 5.6.3 just now!
